pkg/manager/manager.go

```diff
@@ -131,56 +143,82 @@ func (r *LayerManager) ResolverMetaLayer(ctx context.Context, refspec reference.
 		target.Annotations[label.CRIImageRef] = refspec.String()
 		target.Annotations[label.CRILayerDigest] = target.Digest.String()
 
-		workdir := r.nydusFs.UpperPath(rawRef)
-		if _, err := os.Stat(workdir); os.IsNotExist(err) {
+		if _, ok = r.nydusMetaLayer.Load(snapshotID); ok {
+			log.G(ctx).Warnf("nydus duplicate mount meta layer ref is %s digest is %s", refspec.String(), target.Digest.String())
```

Comment:
Ditto, we can use `log.G(ctx).WithError(err).Warnf(...)`

Reply:
there is no error